docs/release-notes.md

```diff
@@ -0,0 +1,69 @@
+**note: do not commit this file, copy and paste the text into the release page**
+
+VMware is enraptured to present version 1.0.0-rc.2 of Contour, our layer 7 HTTP reverse proxy for Kuberentes clusters. As always, without the help of the many community contributors this release would not have been possible. Thank you!
+
+Contour 1.0.0-rc.2 is the second, and hopefully last, release candidate on the path to Contour 1.0.
+
+The current stable release at this time remains Contour 0.15.2.
+
+## New and improved 
+
+Contour 1.0.0-rc.2 contains many bug fixes and improvements over rc.1.
+
+### Website improvements
+
+As part of the continued preparations for the 1.0 release Contour's documentation has been relocated to the https://projectcontour.io website. Specifically;
+
+* The Getting Started documentation has moved to [projectcontour.io/getting-started](https://projectcontour.io/getting-started/)
+* Guides and How-to's have moved to [projectcontour.io/guides](https://projectcontour.io/guides)
+* Versioned release documetaion has moved to [projectcontour.io/docs](https://projectcontour.io/docs)
+* Project related and non-versioned documentation has moved to [projectcontour.io/resources](https://projectcontour.io/resources/)  
+
+### IngressRoute and HTTPProxy status update improvements
+
+IngressRoute and HTTPProxy status updates are now performed by the lead Contour in the deployment. The lead Contour is determined via Kubernetes' standard leader election mechanisms.
+
+If leader election is disabled, all Contours will write status back to the Kubernetes API.
+
+Fixes #1425, #1385, and many more issues with status loops over the years.
+
+### HTTPProxy and IngressRoute OpenAPIv3 schema validation
+
+Contour 1.0.0-rc.2 includes updates the the OpenAPIv3 schema validations. These schemas are automatically generated from the CRD documents themselves and should be more complete and consistent than the previous hand rolled versions.
+
+Fixes #513, #1414. Thanks @youngnick
+
+### Envoy TCP keepalive
```

Comment:
Maybe make this a more general keepalive mention now?

### xDS keepalive tuning

Some issues have been found with connections become Contour and Envoy becoming stuck half-open or half-closed. To address these, we've added two layers of keepalives: TCP keepalives to cover the Envoy side, and gRPC keepalives to cover the Contour side. We've chosen to do both so that both sides of the connection can be confident that open connections really are open.

and then the thanks. Please wordsmith as required.

Reply:
I've redone this with the words from the 0.15.3 release notes.